deployment/test/web.rb

```diff
@@ -1,5 +1,7 @@
 require 'sinatra'
 
+
 get '/' do
-  "Hello, world"
+  whom = File.open("whom.txt").read
```

Comment:
Planning to use this in the deployment test, to actually test a string on the site.